src/items/itemsTable.component.tsx

```diff
@@ -301,7 +291,6 @@ export function ItemsTable(props: ItemTableProps) {
     enableRowVirtualization: false,
     enableFullScreenToggle: false,
     enableColumnVirtualization: dense ? false : true,
-    enableGrouping: !dense,
```

Comment:
Sorry when I said remove grouoping I should have said only for the serial number 😄 please undo this change, and ensure `enableGrouping: false` is added to the serial number column instead. Also undo the changes for created and modified as the grouping will never make sense due to the seconds being included. Asset number is probably ok though.

Reply:
Ohh my bad  😄